app/src/main/java/org/mifos/mobile/viewModels/HomeViewModel.kt

```diff
@@ -26,21 +26,29 @@ class HomeViewModel @Inject constructor(private val homeRepositoryImp: HomeRepos
 
     private val _homeUiState = MutableStateFlow<HomeUiState>(HomeUiState.Loading)
     val homeUiState: StateFlow<HomeUiState> = _homeUiState
-
+    var errorHandled = false
     fun loadClientAccountDetails() {
         viewModelScope.launch {
             _homeUiState.value = HomeUiState.Loading
-            homeRepositoryImp.clientAccounts().catch {
-                _homeUiState.value = HomeUiState.Error(R.string.error_fetching_accounts)
+
+
```

Comment:
remove this extra space

Reply:
done

Reply:
please dont resolve the conversation on your own @Thanush66 .